src/Core/Services/Implementations/LocalSendStorageService.cs

```diff
@@ -58,6 +67,14 @@ private void DeleteFileIfExists(string path)
             }
         }
 
+        private void DeleteDirectoryIfExists(string path)
```

Comment:
Not sure I like the idea of deleting directories and/or storing Send files in a directory per Send.

Reply:
I can revert, not married to the idea. I just liked matching the file structure to Azure. There I have good reasons to want SendId in the blob path.